presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -626,6 +626,50 @@ to bypass the network (*short-circuit*). See `Performance Tuning Tips for Presto
 <https://www.alluxio.io/blog/top-5-performance-tuning-tips-for-running-presto-on-alluxio-1/?utm_source=prestosql&utm_medium=prestodocs>`_
 for more details.
 
+Alluxio Catalog Service
+^^^^^^^^^^^^^^^^^^^^^^^
+
+An alternative way for Presto to interact with Alluxio is via the
+`Alluxio catalog service. <https://docs.alluxio.io/os/user/stable/en/core-services/Catalog.html?utm_source=prestosql&utm_medium=prestodocs>`_.
+The primary benefits for using the Alluxio catalog service are simpler
+deployment of Alluxio with Presto, and enabling schema-aware optimizations
+such as transparent caching and transformations. Currently, the catalog service
+supports read-only workloads.
+
+The Alluxio catalog service is a metastore that can cache the information
+from different underlying metastores. It currently supports the Hive metastore
+as an underlying metastore. In order for the Alluxio catalog to manage the metadata
+of other existing metastores, the other metastores must be "attached" to the
+Alluxio catalog. To attach an existing Hive metastore to the Alluxio
+catalog, simply use the
+`Alluxio CLI attachdb command <https://docs.alluxio.io/os/user/stable/en/operation/User-CLI.html#attachdb?utm_source=prestosql&utm_medium=prestodocs>`_.
+The appropriate Hive metastore location and Hive database name need to be
+provided.
+
+.. code-block:: none
+
+    ./bin/alluxio table attachdb hive thrift://HOSTNAME:9083 hive_db_name
+
+Once a metastore is attached, the Alluxio catalog can manage and serve the
+information to Presto. To configure the Hive connector for Alluxio
+catalog service, simply configure the connector to use the Alluxio
+metastore type, and provide the location to the Alluxio cluster.
+For example, your ``etc/catalog/catalog_alluxio.properties`` includes
```

Comment:
`catalog` in the catalog name seems redundant. We can shorten to `alluxio.properties`